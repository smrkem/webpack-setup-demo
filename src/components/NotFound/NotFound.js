import React from 'react'

const NotFound = () => (
  <div className="container">
    <h1 className="heading">Oops - couldn't find what you were looking for :(</h1>
  </div>
)

export default NotFound
