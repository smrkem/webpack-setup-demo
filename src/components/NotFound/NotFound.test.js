import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import NotFound from './NotFound'

test('NotFound should render ok', () => {
  const component = shallow(<NotFound />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
