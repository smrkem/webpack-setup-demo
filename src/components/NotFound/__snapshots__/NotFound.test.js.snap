// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render ok 1`] = `
<div
  className="container"
>
  <h1
    className="heading"
  >
    Oops - couldn't find what you were looking for :(
  </h1>
</div>
`;
